Source/WebCore/testing/Internals.cpp

```diff
@@ -6827,4 +6827,19 @@ String Internals::modelInlinePreviewUUIDForModelElement(const HTMLModelElement&
 
 #endif
 
+void Internals::shouldBlockIOSurfaceCreationForNextImageBuffer()
```

Comment:
This is not a good name, it looks like a getter but doesn't return anything. If a function doesn't return a bool, it should probably not start with "should" (or "is" or similar prefixes).

Reply:
Changed to `blockIOSurfaceCreationForNextImageBuffer`.